packages/web/src/components/play-bar/desktop/PlayBar.js

```diff
@@ -350,6 +354,10 @@ class PlayBar extends Component {
     const favoriteText = favorited ? messages.unfavorite : messages.favorite
     const repostText = reposted ? messages.reposted : messages.repost
     const matrix = isMatrix()
+    const isPodcast = track?.genre === Genre.PODCASTS ?? false
+    const isNewPodcastControlsEnabled = getFeatureEnabled(
```

Comment:
do we have a useFeatureFlag?

Reply:
would have to convert the component to be a functional component so this was a smaller change